src/Stratis.Bitcoin.Features.SignalR/Events/BlockConnectedClientEvent.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using Stratis.Bitcoin.EventBus;
+using Stratis.Bitcoin.EventBus.CoreEvents;
+
+namespace Stratis.Bitcoin.Features.SignalR.Events
+{
+    public class BlockConnectedClientEvent : IClientEvent
+    {
+        public string Hash { get; set; }
+
+        public int Height { get; set; }
+
+        public Type NodeEventType { get; } = typeof(BlockConnected);
+
+        public void BuildFrom(EventBase @event)
+        {
+            if (@event is BlockConnected blockConnected)
+            {
+                this.Hash = blockConnected.ConnectedBlock.ChainedHeader.HashBlock.ToString();
+                this.Height = blockConnected.ConnectedBlock.ChainedHeader.Height;
+                return;
+            }
+
+            throw new ArgumentException();
```

Comment:
Do we need to throw here?

Reply:
Not my code this is Igors, maybe he can answer this ?

Reply:
Well, if we got to that line then someone is trying to pass invalid event into the `BuildFrom` so technically it is an Agrument exception.

Reply:
I dont think that we need to throw here as this only an eventing service. There isnt critical code that currently depends on this.